src/app/src/test/kotlin/org/imperial/mrc/hint/database/ProjectRepositoryTests.kt

```diff
@@ -224,6 +240,24 @@ class ProjectRepositoryTests
         assertThat(renamedProject[PROJECT.NAME]).isEqualTo("renamedProject")
     }
 
+    @Test
+    fun `can save project note`()
+    {
+        val uid = setupUser()
+
+        val projectId = sut.saveNewProject(uid, "testProjectRepo")
+        val versionId1 = "testVersion"
+        versionRepo.saveVersion(versionId1, projectId)
+
+        sut.saveProjectNote(projectId, "notes", uid)
+
+        val savedNote = dsl.selectFrom(PROJECT)
```

Comment:
```suggestion
        val project = dsl.selectFrom(PROJECT)
```
This is a project, not a note